samples/snippets/src/main/java/com/example/bigtable/deletes/DeleteDropRowRangeExample.java

```diff
@@ -21,12 +21,10 @@
 import java.io.IOException;
 
 public class DeleteDropRowRangeExample {
```

Comment:
should this be DropRowRangeExample or DeleteRowRangeExample? DeleteDrop seems confusing

Reply:
done